### PR TITLE
Require TSC approval for stances

### PR DESCRIPTION
**Pull request by benjamingr**
@nodejs/tsc , bikeshedding/different wording welcome

### Check List

- [ ] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [ ] I have run `npm run format` to ensure the code follows the style guide.
- [ ] I have run `npm run test` to check if all tests are passing.
- [ ] I have run `npx turbo build` to check if the website builds without errors.
- [ ] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

File: GOVERNANCE.md
Comment (ovflowd) [inline GOVERNANCE.md]: Can you also write down what would be considered TSC approval? Adding it to the TSC agenda and getting consensus there? Or, enough thumbs up? I believe we need better instructions here to failsafe this 👀

Comment (marco-ippolito) [reply GOVERNANCE.md]: I think pinging tsc is enough, if there are blockers then they will be discussed

Comment (ovflowd) [reply GOVERNANCE.md]: I think pinging the TSC is a pre-requisite. But what would he a TSC approval? No objections? One thumbs up? And for how long should we wait for the feedback of the TSC?

Comment (aduh95) [reply GOVERNANCE.md]: I think the process should be "sending an email to tsc@iojs.org and receiving no objections and at least one approval in seven days". I don't think the TSC meetings would be a good place to discuss this, it's probably better to have this sort of discussions in private (TSC members might not all be in a position where they can express publicly political opinions)

Comment (ovflowd) [reply GOVERNANCE.md]: Im not sure how I feel about the TSC making such actions behind the scenes (through a private email) whereas the Website Team or the author of the contribution gets publicly exposed for idealising a patch that might contain a political statement.

I understand the want of anonymity but I believe it makes sense for at least one member of the TSC publicly express the approval. Otherwise it's just me saying "ah yes the TSC approved it" ...

Comment (benjamingr) [reply GOVERNANCE.md]: TSC approval is a documented process. The TSC follows the consensus seeking process and if it can't built consensus the TSC votes.

Comment (ovflowd) [reply GOVERNANCE.md]: Is this what Antoine commented, the official TSC approval process? 

Well, there's then very little I can do with that.

Comment (aduh95) [reply GOVERNANCE.md]: On a related note, we have precedence for authoring a commit from the project (https://github.com/nodejs/nodejs.org/commit/a87f468982dde3e5931f4c3a2ee7c67cc6253e67), so no individual needed to take "the blame", although I'm not sure if that's what you're asking.
I think if some TSC members want to express publicly their approval, that's certainly fine, but we should not require it. If the website team is not comfortable moving forward if there are no public approvals, that's certainly fine, but I don't think it needs to be in the policy. Does that make sense?

Comment (ovflowd) [reply GOVERNANCE.md]: That's fair! Thanks for explaining, Antoine!

**Review by ovflowd — APPROVED** [review]
For all it's worth, I appreciate the TSC proactively handling this.

**Review by mcollina — APPROVED** [review]
lgtm

File: GOVERNANCE.md
Comment (bmuenzenmeyer) [inline GOVERNANCE.md]: Typo/grammar

```suggestion
Any website change that expresses a position about a global event or group of people requires explicit [TSC](https://github.com/nodejs/TSC/blob/main/TSC-Charter.md#section-4-responsibilities-of-the-tsc) approval.
```

**Review by tniessen — APPROVED** [review]

**Review by marco-ippolito — APPROVED** [review]

**Comment by bmuenzenmeyer** [comment]
Does anyone have strong opinions about how this relates to [CONTENT VS CODE](https://github.com/nodejs/nodejs.org/blob/main/CONTENT_VS_CODE.md)? 

I think if I had to pick, I'd see it as content. Reading those guidelines in accordance with this change, I'd resolve to say it re-enforces TSC involvement. Perhaps they should be cross-linked.

**Review by bmuenzenmeyer — APPROVED** [review]

**Comment by ovflowd** [comment]
> Does anyone have strong opinions about how this relates to [CONTENT VS CODE](https://github.com/nodejs/nodejs.org/blob/main/CONTENT_VS_CODE.md?rgh-link-date=2024-06-02T19%3A39%3A18Z)?
> 
> I think if I had to pick, I'd see it as content. Reading those guidelines in accordance with this change, I'd resolve to say it re-enforces TSC involvement. Perhaps they should be cross-linked.

cc @nodejs/tsc

**Review by ShogunPanda — APPROVED** [review]

**Review by AugustinMauroy — APPROVED** [review]

**Review by BridgeAR — APPROVED** [review]

**Comment by ovflowd** [comment]
Standard governance changes within this repository require an extended period for approval. But I can't imagine the TSC or the Web Team would be against this change. To be honest, we can merge this as soon as possible. But I'm also fine waiting the 72 hours.

I'd like @bmuenzenmeyer's concern addressed before we merge this, just so we can easily clarify/connect both documents, which serve as a guiding mechanism.

Is the TSC OK if we update the docs of CONTENT_VS_CODE to match governance changes?

**Review by aymen94 — APPROVED** [review]

**Comment by benjamingr** [comment]
>. Standard governance changes within this repository require an extended period for approval. But I can't imagine the TSC or the Web Team would be against this change. To be honest, we can merge this as soon as possible. But I'm also fine waiting the 72 hours.

Please do not merge this until 48h have passed or at least TSC members have had time to look when back from the weekend.

**Review by targos — APPROVED** [review]

File: GOVERNANCE.md
Comment (aduh95) [inline GOVERNANCE.md]: Let's clarify what "TSC approval" means
```suggestion
Any website change that expresses a position about a global event or group of people requires explicit
[TSC](https://github.com/nodejs/TSC/blob/main/TSC-Charter.md#section-4-responsibilities-of-the-tsc)
approval. This can be obtained by pinging `@nodejs/tsc` and receive no objections after seven days,
or by sending an email to `tsc@iojs.org` and receive at least one approval and no objections after seven days.
```

**Review by MoLow — APPROVED** [review]

**Review by ruyadorno — APPROVED** [review]

**Review by legendecas — APPROVED** [review]

**Review by richardlau — APPROVED** [review]

**Review by joyeecheung — APPROVED** [review]

**Review by jasnell — APPROVED** [review]

**Comment by jasnell** [comment]
I think it should likely be stressed that this is not a gatekeeping measure meant to discourage these kinds of changes but more of an awareness thing. Specifically, the TSC needs to be given the time to be aware that such changes are proposed/pending and given the time to weigh in on the possible ramifications of the change.

**Comment by benjamingr** [comment]
@jasnell I've tried to use as neutral objective language in the text but feel free to suggest edits if you think it implies a bias against or for such initiatives.

**Comment by avivkeller** [comment]
IMO a variant of this should be added to <https://github.com/nodejs/node/blob/main/GOVERNANCE.md#technical-steering-committee> and <https://github.com/nodejs/TSC#list-of-tsc-responsibilities>

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by jasnell** [comment]
I'm not suggesting that any changes be made to the proposed text (otherwise I would have made that explicit)... I just wanted to stress the point in the PR discussion here.

**Comment by bmuenzenmeyer** [comment]
48 hours have passed. appreciate the strong alignment on this one

**Pull request merged**
